code/game/objects/items/crayons.dm

```diff
@@ -739,6 +739,12 @@
 		user.visible_message("<span class='danger'>[user] sprays [src] into the face of [target]!</span>")
 		to_chat(target, "<span class='userdanger'>[user] sprays [src] into your face!</span>")
 
+		if(user == target) // huffin spray
```

Comment:
should probably also check to make sure that you can breathe the stuff in

Reply:
make it require target the mouth so you can still spray your own face if you want

Reply:
you're spraying paint onto your unprotected face, you're gonna get high off it